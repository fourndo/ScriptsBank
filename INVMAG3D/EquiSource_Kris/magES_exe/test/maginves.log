
Parallelized with OpenMP.  # of threads:  4

MAGES - Version 1.0 :  MAGINVES

Developed by University of British Columbia 
Geophysical Inversion Facility  (UBC-GIF)

(C) Copyright 1992 - 2014, UBC-GIF,
Department of Earth and Ocean Sciences, UBC
http://gif.eos.ubc.ca/


Reading input file: inv.inp

----------------------------------------------------------------

1
1, 0.02
obs.mag
maginves.mtx
-10.0
10.0
100 100
null !flat_full.mtx

----------------------------------------------------------------


MAGINVES started on:  3/20/2014  13:35:07


Running with the following parameters:

           Input file: inv.inp
                 mode: 1  (fixed target misfit)
              chifact:   1.00    
    Observations file: obs.mag
     Sensitivity file: maginves.mtx
Full Sensitivity file: null


                    # of data:     1102

                   # of cells: 109 x 86
             total # of cells:    9374
        # of cells below topo:    9374

      lower bounds was set to: -1.0000E+01
      upper bounds was set to:  1.0000E+01


----------------------------------------
 length scales:
                 Le, Ln:   100.      100.    
 corresponding alpha values:
        alpha (s, e, n):  1.000E-04  1.00      1.00    
----------------------------------------

      init cpu time:   0:00:00.10
 
     initial misfit:  1.474862E+05
 target data misfit:  1.102000E+03

          Iteration:   1
               beta:  4.107381E+10
        data misfit:  6.906873E+04
      beta cpu time:   0:00:01.57

          Iteration:   2
               beta:  2.053691E+10
        data misfit:  6.898256E+04
      beta cpu time:   0:00:01.57

          Iteration:   3
               beta:  1.026845E+10
        data misfit:  6.873178E+04
      beta cpu time:   0:00:01.25

          Iteration:   4
               beta:  5.134227E+09
        data misfit:  6.824180E+04
      beta cpu time:   0:00:01.27

          Iteration:   5
               beta:  2.567113E+09
        data misfit:  6.730464E+04
      beta cpu time:   0:00:01.16

          Iteration:   6
               beta:  1.283557E+09
        data misfit:  6.557943E+04
      beta cpu time:   0:00:01.38

          Iteration:   7
               beta:  6.417783E+08
        data misfit:  6.259531E+04
      beta cpu time:   0:00:01.31

          Iteration:   8
               beta:  3.208892E+08
        data misfit:  5.786968E+04
      beta cpu time:   0:00:01.16

          Iteration:   9
               beta:  1.604446E+08
        data misfit:  5.113427E+04
      beta cpu time:   0:00:01.37

          Iteration:  10
               beta:  8.022229E+07
        data misfit:  4.252507E+04
      beta cpu time:   0:00:01.45

          Iteration:  11
               beta:  4.011115E+07
        data misfit:  3.266219E+04
      beta cpu time:   0:00:01.83

          Iteration:  12
               beta:  2.005557E+07
        data misfit:  2.263934E+04
      beta cpu time:   0:00:02.65

          Iteration:  13
               beta:  1.002779E+07
        data misfit:  1.379013E+04
      beta cpu time:   0:00:02.57

          Iteration:  14
               beta:  5.013893E+06
        data misfit:  7.181225E+03
      beta cpu time:   0:00:02.80

          Iteration:  15
               beta:  2.506947E+06
        data misfit:  3.133918E+03
      beta cpu time:   0:00:03.45

          Iteration:  16
               beta:  1.253473E+06
        data misfit:  1.148804E+03
      beta cpu time:   0:00:05.62

   Target misfit reached.



     TOTAL cpu time:   0:00:32.60

MAGINVES ended on:  3/20/2014  13:35:39

