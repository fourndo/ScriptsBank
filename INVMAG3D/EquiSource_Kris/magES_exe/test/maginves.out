
! MAGINVES started on:  3/20/2014  13:35:07

!    1  1.0000   !! mode, chifact
!   1.2593E-01   !! initial norm S
!   0.0000E+00   !! initial norm E
!   0.0000E+00   !! initial norm N
!   1.2593E-01   !! initial norm total
!   1.4749E+05   !! initial misfit
!   1.1020E+03   !! target data misfit
!  1.000E-04  1.00      1.00    !! alpha v

! iter          beta   data misfit        norm S        norm E        norm N    model norm  total object
     1  4.107381E+10  6.906873E+04  1.833412E-08  1.195776E-08  9.254371E-09  3.954625E-08    7.069305E+04            51             0
     2  2.053691E+10  6.898256E+04  8.305819E-10  2.442796E-09  2.911037E-09  6.184415E-09    6.910957E+04            47             0
     3  1.026845E+10  6.873178E+04  3.300344E-09  9.639266E-09  1.154442E-08  2.448403E-08    6.898320E+04            36             0
     4  5.134227E+09  6.824180E+04  1.300153E-08  3.754221E-08  4.539397E-08  9.593772E-08    6.873437E+04            37             0
     5  2.567113E+09  6.730464E+04  5.049402E-08  1.426673E-07  1.756867E-07  3.688481E-07    6.825151E+04            36             0
     6  1.283557E+09  6.557943E+04  1.911824E-07  5.189911E-07  6.607714E-07  1.370945E-06    6.733911E+04            37             0
     7  6.417783E+08  6.259531E+04  6.939230E-07  1.759563E-06  2.369633E-06  4.823119E-06    6.569068E+04            37             0
     8  3.208892E+08  5.786968E+04  2.365877E-06  5.410553E-06  7.919692E-06  1.569612E-05    6.290639E+04            39             0
     9  1.604446E+08  5.113427E+04  7.426908E-06  1.490942E-05  2.416357E-05  4.649990E-05    5.859492E+04            42             0
    10  8.022229E+07  4.252507E+04  2.110076E-05  3.719089E-05  6.647083E-05  1.247625E-04    5.253380E+04            46             0
    11  4.011115E+07  3.266219E+04  5.347590E-05  8.553220E-05  1.639222E-04  3.029303E-04    4.481308E+04            53             0
    12  2.005557E+07  2.263934E+04  1.195894E-04  1.825817E-04  3.601894E-04  6.623605E-04    3.592336E+04            65             0
    13  1.002779E+07  1.379013E+04  2.346044E-04  3.573052E-04  6.995119E-04  1.291421E-03    2.674023E+04            78             0
    14  5.013893E+06  7.181225E+03  4.025165E-04  6.258036E-04  1.192735E-03  2.221055E-03    1.831736E+04           101             0
    15  2.506947E+06  3.133918E+03  6.033831E-04  9.603629E-04  1.781791E-03  3.345537E-03    1.152100E+04           127             0
    16  1.253473E+06  1.148804E+03  7.964050E-04  1.288955E-03  2.348936E-03  4.434296E-03    6.707075E+03           175             0

! MAGINVES ended on:  3/20/2014  13:35:39

